underfs/obs/src/main/java/alluxio/underfs/obs/OBSPropertyKey.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.obs;
+
+import alluxio.conf.PropertyKey;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * OBS configuration property keys.
+ */
+@ThreadSafe
+public final class OBSPropertyKey {
```

Comment:
Please move all the property key into PropertyKey file and delete this file.
This file is used in ufs-extensions but not needed when move back to Alluxio

Reply:
done